sample.env

```diff
@@ -1,6 +1,10 @@
-DOCKER_HOST_IP=10.1.2.3
-#SDW_GROUP_ID=usdot.jpo.sdw
-#SDW_KAFKA_PORT=9092
-#SDW_DESTINATION_URL=https://webapp-integration.cvmvp.com/whtools/rest/v2/
-SDW_SUBSCRIPTION_TOPIC=<your topic to subscribe to for deposit>
```

Comment:
Some of these seem like we should keep them in, for instance the sdw_destination_url, sdw_kafka_port (for overrides), sdw_group_id (when spinning up multiple this will be required).

Reply:
Environment variables that were previously removed have been returned to this file.